stories/timeline/props-table.tsx

```diff
@@ -102,18 +102,73 @@ const DATA: ApiTableSectionProps[] = [
     {
         name: "TimelineStatusProps",
         attributes: [
+            {
+                name: "",
+                description: (
+                    <>
+                        This component also inherits props from&nbsp;
+                        <a
+                            href="https://developer.mozilla.org/en-US/docs/Web/API/HTMLDivElement"
+                            rel="noreferrer"
+                            target="_blank"
+                        >
+                            HTMLDivAttributes
+                        </a>
+                    </>
+                ),
+            },
             {
                 name: "type",
                 mandatory: true,
-                description: "The color style of the pill",
-                propTypes: [`"dark"`, `"light"`],
+                description: (
+                    <>
+                        The color style of the pill.{" "}
+                        <b>
+                            Note: from v2.4.0 onwards, values `dark`` and
```

Comment:
There's an extra back tick after dark. Anyway i suggest to add a `<br />` for the note. Then use the `code` helper function to style the prop values. 
```
code(`"dark"`)
```